packages/sshnoports/lib/sshnp/sshnp_impl.dart

```diff
@@ -638,6 +638,7 @@ class SSHNPImpl implements SSHNP {
         ..key = 'sshpublickey'
         ..sharedBy = clientAtSign
         ..sharedWith = sshnpdAtSign
+        ..namespace = namespace
```

Comment:
In 4.0.0-rc2 I see this in daemon logs
```
INFO|2023-09-08 14:05:31.280427| sshnpd |ssh Public Key received from @garycasey notification id : d5e36215-aee7-4682-8a5a-c77b8e83e6f2 
```
and the key is successfully added to the authorized_keys file

so I don't believe this is currently a bug, because the namespace is picked up from the AtClient's namespace.

However it is indeed better to be explicit and set the namespace